Core/src/org/sleuthkit/autopsy/casemodule/Case.java

```diff
@@ -1523,7 +1523,7 @@ private static void doCaseChange(Case toChangeTo) {
                 CallableSystemAction.get(CaseCloseAction.class).setEnabled(true);
                 CallableSystemAction.get(CasePropertiesAction.class).setEnabled(true);
                 CallableSystemAction.get(CaseDeleteAction.class).setEnabled(true); // Delete Case menu
-
+                CallableSystemAction.get(OpenTimelineAction.class).setEnabled(true);
                 if (toChangeTo.hasData()) {
```

Comment:
Looking at this, it seems strange that the enabling of the actions is not being dispatched to the EDT queue with SwingUtilities.invokeLater. Please do so, as for the case closed branch of the conditional.